src/main.js

```diff
@@ -55,21 +55,20 @@ let loud = false;
 if (typeof window !== "undefined") {
   runtimeConfig = paramsToObj(window.location.search);
   loud = runSafe = !!window.location.search; // anything passed!
+  runSafe && personinfo.tryLocalStorage();  // 1st attempt, 2nd during mainloop.  see #212
 }
 
-
-// will only catch things from other windows
-//if (window) {
-//  window.addEventListener("storage", function (e) {
-//    log("storage", key, oldValue, newValue, uri);
-//  }, false);
-//}
-
-
 // is there a timer here? I dunno!
 let mainloop = function (repairsList) {
   loud && actions.log("heartbeat main loop");
-  runner.runAll(repairsList,
+  personinfo.tryLocalStorage();  // see #212.
```

Comment:
So... I'm confused. The comments here specify that you're calling `tryLocalStorage()` twice in the same run, ie once in global scope if `typeof window !== "undefined"`, and once in `mainloop`. That means that after those, the check variable in `localStorage` will have been incremented to 2, right?